<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneSizeInputClass</class>
 <widget class="QDialog" name="SceneSizeInputClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scene Dimensions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>WARNING: This will reset the graph.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter scene dimensions:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="widthBox">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>1280</number>
     </property>
     <property name="maximum">
      <number>122880</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="heightBox">
     <property name="minimum">
      <number>720</number>
     </property>
     <property name="maximum">
      <number>69120</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Update Scene Size</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
